cmd/registry/controller/controller.go

```diff
@@ -68,7 +67,7 @@ func ProcessManifest(
 
 func processManifestResource(
 	ctx context.Context,
-	client connection.RegistryClient,
+	client Lister,
```

Comment:
Could "client" be "lister" instead? I think that would be more clear.

Reply:
My intention of naming it client is to indicate that the functionality is still like a client to talk to the registry, but a specific implementation of the client: `Lister`, which focuses on `List` calls.

However, I have used this as `listerClient` in some parts of the code. For consistency, I am going to name everything as client.

Reply:
Could we put `Client` in the type name of `Lister`? As it is, these terms are so conceptually distant that they introduce confusion to new readers of the code. `ListingClient` might resolve this.

Reply:
I agree.